pkgs/development/compilers/lessc/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     description = "LESS to CSS compiler";
     homepage = http://lesscss.org/;
     license = licenses.asl20;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
Is there a reason why this shouldn't work on bsd systems? Otherwise you should use `platforms.unix`